### PR TITLE
Feature/apache beam samples

### PR DESCRIPTION
**Pull request by mmistroni**
# Pull Request Template for OpenBB Developers

0. **Title**:

    - Sample showing how to call OBB code from Apache Beam task

1. **Why**? (1-3 sentences or a bullet point list):

    - Useful to show some automation as OBB code can be ran by subrocesses running on a pipeline."

2. **What**? (1-3 sentences or a bullet point list):

    -  Have created a simple unit test that ran an OBB Fetcher within an Apache Beam pipeline

3. **Impact** (1-2 sentences or a bullet point list):

    -N.A

4. **Testing Done**:

    - The code ran in a unit test

### CODE REVIEW

**Comment by deeleeramone** [comment]
This is cool, thanks for the PR. I still need to do my homework to be able to fully test drive this puppy, so correct me if I'm wrong, I believe we can simplify the requirements to just `openbb-yfinance`, and we wouldn't even need a version number. Can you confirm? You shouldn't have to change any code for that, and it will shrink the overall footprint of the environment.

**Comment by mmistroni** [comment]
Yes that is right..I just wanted to get whole obb to make sure code works
Rgds

**Review by deeleeramone — COMMENTED** [review]
Thanks a lot for this unique example, I added a few minor comments and one question.

I followed the instructions and everything worked as-expected.

File: examples/openbb-apachebeam/README.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#### Obb Dataflow sample
+
+
+This is a sample how to  invoke OBB fetchers in an Apache Beam pipeline. (GCP dataflow is build on Apache Beam)
+-- Pre-requisite
+# You need to create a Conda environment (or a virtual env) using requirements.txt in this directory
+# The script exercise 3 OBB endpoints, all of which require no credentials
+# Run the test from the main directory
+  (obb-dataflow) C:\Users\Marco And Sofia\tmp\OBBDataflowSample>python -m unittest discover
```
[the remark this answers is not in the document]
Comment (mmistroni) [reply examples/openbb-apachebeam/README.txt:9]: pytests on windows, with discover, automatically find the unit test,. its just me being lazy and wanted to keep setup as minimal as possible, i'll amend instr

File: examples/openbb-apachebeam/README.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#### Obb Dataflow sample
+
+
+This is a sample how to  invoke OBB fetchers in an Apache Beam pipeline. (GCP dataflow is build on Apache Beam)
+-- Pre-requisite
```
[the remark this answers is not in the document]
Comment (mmistroni) [reply examples/openbb-apachebeam/README.txt:5]: ok

**Comment by deeleeramone** [comment]
Closing because reopened as #6679

**Pull request closed without merging**